### ISSUE TITLE
about the Dropout

### ISSUE THREAD

**Issue by ucasiggcas** [tensorflow/tensorflow#51593]
hi,dear all
when use Dropout as the **help** down, I got confused, **what sum is not changed** ??
```
>>> help(tf.keras.layers.Dropout)
class Dropout(tensorflow.python.keras.engine.base_layer.Layer)
 |  Dropout(*args, **kwargs)
 |  
 |  Applies Dropout to the input.
 |  
 |  The Dropout layer randomly sets input units to 0 with a frequency of `rate`
 |  at each step during training time, which helps prevent overfitting.
 |  Inputs not set to 0 are scaled up by 1/(1 - rate) such that the **sum** over
 |  all inputs is unchanged.
```

```
>>> layer = tf.keras.layers.Dropout(.2, input_shape=(2,))
>>> data = np.arange(10).reshape(5, 2).astype(np.float32)
>>> data
array([[0., 1.],
       [2., 3.],
       [4., 5.],
       [6., 7.],
       [8., 9.]], dtype=float32)
>>> outputs = layer(data, training=True)
>>> outputs
<tf.Tensor: shape=(5, 2), dtype=float32, numpy=
array([[ 0.  ,  1.25],
       [ 2.5 ,  3.75],
       [ 5.  ,  6.25],
       [ 7.5 ,  0.  ],
       [10.  , 11.25]], dtype=float32)>
>>> np.sum(outputs)
47.5
>>> np.sum(data)
45.0
>>> np.sum(data)/10
4.5
>>> np.sum(data)/8
5.625
>>> np.sum(data,axis=1)
array([ 1.,  5.,  9., 13., 17.], dtype=float32)
>>> np.sum(outputs,axis=1)
array([ 1.25,  6.25, 11.25,  7.5 , 21.25], dtype=float32)
>>> np.sum(outputs,axis=0)
array([25. , 22.5], dtype=float32)
>>> np.sum(data,axis=0)
array([20., 25.], dtype=float32)
>>> np.sum(outputs)/8
5.9375
>>> np.sum(outputs)/10
4.75

```
could you pls help me ?
thx

**Comment by ucasiggcas** [comment]
and how to set the para **input_shape** ? if I set the input_shape as input, got similar result, 
```
>>> layer = tf.keras.layers.Dropout(.2, input_shape=(5,2))
>>> outputs = layer(data, training=True)
>>> outputs
<tf.Tensor: shape=(5, 2), dtype=float32, numpy=
array([[ 0.  ,  1.25],
       [ 2.5 ,  3.75],
       [ 0.  ,  6.25],
       [ 7.5 ,  0.  ],
       [ 0.  , 11.25]], dtype=float32)>
```

**Comment by sushreebarsa** [comment]
@ucasiggcas Please post this issue on [keras-team/keras repo.](https://github.com/keras-team/keras/issues)
To know more see;
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999)
Thank you!
